src/webextension/list/manage/components/account-summary-label.js

```diff
@@ -23,4 +27,85 @@ export default function AccountSummaryLabel({displayName, avatar}) {
 AccountSummaryLabel.propTypes = {
   displayName: PropTypes.string,
   avatar: PropTypes.string,
+  onClick: PropTypes.func,
 };
+
+function AccountSummaryDropdown({isOpen, onClose, onClickAccount,
+                                 onClickOptions, onClickSignout}) {
+  return (
+    <Modal isOpen={isOpen} onRequestClose={onClose}
+           className={styles.dropdown}
+           overlayClassName={styles.dropdownOverlay}>
+      <ul>
+        <li>
+          <Localized id="account-summary-account">
+            <Link onClick={() => { onClickAccount(); onClose(); }}>
+              aCCOUNt
+            </Link>
+          </Localized>
+        </li>
+        <li>
+          <Localized id="account-summary-options">
+            <Link onClick={() => { onClickOptions(); onClose(); }}>
+              oPTIONs
+            </Link>
+          </Localized>
+        </li>
+        <li>
+          <Localized id="account-summary-signout">
+            <Link onClick={() => { onClickSignout(); onClose(); }}>
```

Comment:
@linuxwolf I lean towards it happening via the message handler, since signing out from *anywhere* (say, the doorhanger), should close the management tab, right?

Reply:
that makes the most sense, yes.

Reply:
Ok, this should be working via the message handler now. (Though I do need to actually test it...)